user_manual/files/desktop_mobile_sync.rst

```diff
@@ -14,11 +14,10 @@ computer, it will flow across the others using these desktop sync clients.
 You will always
 have your latest files with you wherever you are.
 
-Its usage is documented separately in the `Nextcloud/ownCloud Desktop Client Manual`_.
+Its usage is documented separately in the `Nextcloud Desktop Client Manual`_.
 
-.. _`Nextcloud/ownCloud Desktop Client Manual`: https://doc.owncloud.org/desktop/2.2/
-.. _Nextcloud Desktop Client Manual: https://docs.nextcloud.org/
-.. _Nextcloud Sync Client: https://nextcloud.com/install/#install-clients
+.. _`Nextcloud/ownCloud Desktop Client Manual`: https://docs.nextcloud.org/desktop/
```

Comment:
docs.nextcloud.*com* instead of *.org* ... Link is depending on #678

Reply:
good catch!

Reply:
Also it needs to be "Nextcloud Desktop Client Manual" because this is a named reference from line 17 ;)